docs/general-usage/hooks.rst

```diff
@@ -31,4 +35,67 @@ replace or remove any of the default mixins that are not of use in your project:
     @hooks.register('register_schema_query')
     def replace_search_query(query_mixins):
         query_mixins[:] = [item for item in query_mixins if item != SearchQuery]
-        quer_mixins.append(ReplacementQuery)
+        query_mixins.append(ReplacementQuery)
+
+Mutation
+^^^^^^^^
+
+Grapple provides a ``register_schema_mutation`` hook that is called when it creates the schema. You can use it to add your custom ``Mutation`` mixins
+
+.. code-block:: python
+    # mutations.py
+    class CreateAuthor(graphene.Mutation):
+        class Arguments:
+            name = graphene.String()
+            parent = graphene.Int()
+
+        ok = graphene.Boolean()
+        author = graphene.Field(
+            PageInterface,
+        )
+
+        def mutate(root, info, name, parent):
+            author = AuthorPage(name=name, title=name, slug=name)
+            ok = True
+            Page.objects.get(id=parent).add_child(instance=author)
+            author.save_revision().publish()
+            return CreateAuthor(author=author, ok=ok)
+
+
+    class Mutation(graphene.ObjectType):
+        create_author = CreateAuthor.Field()
+
+
+.. code-block:: python
+    # wagtail_hooks.py
+    from .mutations import Mutation
+
+    @hooks.register("register_schema_mutation")
+    def register_author_mutation(mutation_mixins):
+        mutation_mixins.append(Mutation)
+
+
+Subscription
+^^^^^^^^^^^^
+
+Grapple provides a ``register_schema_subscription`` hook that is called when it creates the schema. You can use it to add your custom ``Subscription`` mixins
```

Comment:
Worth making a note that this is enabled only when installed with `wagtail_grapple[channels]`